cmd/api/src/api/v2/saved_queries_test.go

```diff
@@ -262,6 +263,321 @@ func TestResources_ListSavedQueries(t *testing.T) {
 	}
 }
 
+func TestResources_ListSavedQueries_OwnedQueries(t *testing.T) {
+	var (
+		mockCtrl  = gomock.NewController(t)
+		mockDB    = mocks.NewMockDatabase(mockCtrl)
+		resources = v2.Resources{DB: mockDB}
+	)
+	defer mockCtrl.Finish()
+
+	endpoint := "/api/v2/saved-queries"
+	userId, err := uuid2.NewV4()
+	require.Nil(t, err)
+
+	mockDB.EXPECT().ListSavedQueries(gomock.Any(), userId, gomock.Any(), gomock.Any(), 1, 10).Return(model.SavedQueries{
+		{
+			UserID:      userId.String(),
+			Name:        "myQuery",
+			Query:       "Match(n) return n;",
+			Description: "Public query description",
+		},
+	}, 1, nil)
+
+	if req, err := http.NewRequestWithContext(createContextWithOwnerId(userId), "GET", endpoint, nil); err != nil {
```

Comment:
If we change these to:

```go
req, err := http.NewRequestWithContext(createContextWithOwnerId(userId), "GET", endpoint, nil)
require.Nil(t, err)

q := url.Values{}
...
```

We can avoid indenting logic flows and better see what points are required to run the test

Reply:
updated!